### PR TITLE
[release-0.26] include go1.17 build tags & bump workflows

### PR DESCRIPTION
**Pull request by dprotaso**
unblocks - https://github.com/knative/pkg/pull/2441

### CODE REVIEW

File: .github/workflows/knative-style.yaml [hunk 1]
```diff
@@ -119,16 +130,6 @@ jobs:
...
 
-      - id: golangci_configuration
-        uses: andstor/file-existence-action@v1
-        with:
-          files: .golangci.yaml
-      - name: Go Lint
-        if: steps.golangci_configuration.outputs.files_exists == 'true'
-        uses: golangci/golangci-lint-action@v2
-        with:
-          version: v1.42
```
Comment (mattmoor) [inline .github/workflows/knative-style.yaml:130]: Did you mean to drop this?

Comment (dprotaso) [reply .github/workflows/knative-style.yaml:130]: I blindly copied the latest actions from `knative-sandbox/.github` so I guess it was dropped there?

File: .github/workflows/knative-verify.yaml [hunk 2]
```diff
@@ -45,22 +41,16 @@ jobs:
 
     - name: Install Dependencies
       run: |
-        curl -L https://github.com/google/ko/releases/download/v0.6.0/ko_0.6.0_Linux_x86_64.tar.gz | tar xzf - ko
-        chmod +x ./ko
-        sudo mv ko /usr/local/bin
-        go get github.com/google/go-licenses
+        go install github.com/google/ko@latest
```
Comment (mattmoor) [inline .github/workflows/knative-verify.yaml:44]: Jason Hall has an action to install a release of `ko`.  Installing from source takes like 60s FYI.

**Review by mattmoor — APPROVED** [review]

**Comment by dprotaso** [comment]
@mattmoor PTAL again

**Review by mattmoor — COMMENTED** [review]
/lgtm

**Pull request merged**